Gemfile

```diff
@@ -1,2 +1,2 @@
-source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
\ No newline at end of file
+source 'http://rubygems.org'
```

Comment:
Is there a reason why https was changed to http here?

Reply:
Yes, the reason is, that most OS don't trust the certificate and installing it may not be easy for everyone.

Reply:
Thanks for the info, didn't know that.